interface/wx/ribbon/bar.h

```diff
@@ -462,6 +462,57 @@ class wxRibbonBar : public wxRibbonControl
     */
     wxRibbonDisplayMode GetDisplayMode() const;
 
+    /**
+        Returns whether the bar contains multiple pages.
+
+        The default implementation always returns @c true.
+
+        @since 3.1.0
```

Comment:
This function seems to exist since the very first wxRibbon version.

Reply:
Fixed